react/features/analytics/functions.js

```diff
@@ -109,10 +129,10 @@ export function initAnalytics({ getState }: { getState: Function }) {
  * successfully loaded and rejects if there are no handlers loaded or the
  * analytics is disabled.
  */
-function _loadHandlers(scriptURLs, handlerConstructorOptions) {
+function _loadHandlers(scriptURLs = [], handlerConstructorOptions) {
     const promises = [];
 
-    for (const url of scriptURLs) {
+    scriptURLs.forEach(url => {
```

Comment:
why?

Reply:
Personally prefer forEach and since I was there I changed it. If you don't like it I will remove it...

Reply:
I'm confused here. Do you want `forEach` or `for .. of`?

Are you saying we have a convention to use `for .. of` instead of `forEach`?

Reply:
> Are you saying we have a convention to use `for .. of` instead of `forEach`?

Yes.

Reply:
This is the first time I'm hearing about this and I personally disagree with this policy!

I will revert to the change to use `for .. of`!

Let's discuss the policy offline! I want to hear more about it!